### PR TITLE
Correction of the inability to translate some fields into other languages

### PR DESCRIPTION
**Pull request by MrWeez**

### CODE REVIEW

File: themes/default/views/servers/settings.blade.php [hunk 1]
```diff
@@ -75,7 +75,7 @@
                       <div class="row">
                         <div class="col-8">
                           <div class="numbers">
-                            <p class="text-sm mb-0 text-uppercase font-weight-bold">Memory</p>
+                            <p class="text-sm mb-0 text-uppercase font-weight-bold">{{ __('MEMORY') }}</p>
                             <h5 class="font-weight-bolder">
                               <span class="text-success text-sm font-weight-bolder">@if($server->product->memory == 0)Unlimited @else {{$server->product->memory}}MB @endif</span>
```
Comment (AGuyNamedJens) [inline themes/default/views/servers/settings.blade.php:80]: If you're adding more translations i'd suggest translating the other words such as Unlimited as well :)

File: themes/default/views/servers/settings.blade.php [hunk 2]
```diff
@@ -96,7 +96,7 @@
                       <div class="row">
                         <div class="col-8">
                           <div class="numbers">
-                            <p class="text-sm mb-0 text-uppercase font-weight-bold">STORAGE</p>
+                            <p class="text-sm mb-0 text-uppercase font-weight-bold">{{ __('STORAGE') }}</p>
                             <h5 class="font-weight-bolder">
                               <span class="text-success text-sm font-weight-bolder">@if($server->product->disk == 0)Unlimited @else {{$server->product->disk}}MB @endif</span>
```
Comment (AGuyNamedJens) [inline themes/default/views/servers/settings.blade.php:101]: same as above

File: themes/default/views/servers/settings.blade.php [hunk 3]
```diff
@@ -54,7 +54,7 @@
                       <div class="row">
                         <div class="col-8">
                           <div class="numbers">
-                            <p class="text-sm mb-0 text-uppercase font-weight-bold">CPU</p>
+                            <p class="text-sm mb-0 text-uppercase font-weight-bold">{{ __('CPU') }}</p>
                             <h5 class="font-weight-bolder">
                               <span class="text-success text-sm font-weight-bolder">@if($server->product->cpu == 0)Unlimited @else {{$server->product->cpu}} % @endif</span>
```
Comment (AGuyNamedJens) [inline themes/default/views/servers/settings.blade.php:59]: same as the other 2

**Review by AGuyNamedJens — CHANGES_REQUESTED** [review]
You're forgetting some translations

**Comment by MrWeez** [comment]
> You're forgetting some translations

Thanks, I didn't notice right away. Fixed.

**Comment by MrWeez** [comment]
> Incorrect syntax, I'll fix it now

Although no, that's right. I just checked the code on the panel from the main branch, forgetting that the changes were made in development

**Review by AGuyNamedJens — COMMENTED** [review]
Not sure if you've added the translations to their language files yet, otherwise looks good to me.

**Comment by MrWeez** [comment]
> Not sure if you've added the translations to their language files yet, otherwise looks good to me.

I could upload the localization file of my panel (ru.json), but there are a lot of localized translation moments

**Comment by AGuyNamedJens** [comment]
> > Not sure if you've added the translations to their language files yet, otherwise looks good to me.
> 
> I could upload the localization file of my panel (ru.json), but there are a lot of localized translation moments

https://github.com/Ctrlpanel-gg/panel/blob/development/lang just add the variables to these files, use google translate if you don't know all language variants for them.

**Comment by MrWeez** [comment]
I am not responsible for the accuracy of the translation, but I seem to have added it everywhere)

**Review by AGuyNamedJens — APPROVED** [review]

**Comment by MrWeez** [comment]
After the release of 0.10, I will most likely completely rework the Russian language, since now it looks very illiterate and at times it is not correct at all.

**Comment by AGuyNamedJens** [comment]
> After the release of 0.10, I will most likely completely rework the Russian language, since now it looks very illiterate and at times it is not correct at all.

Sounds logical, as the current contributors don't speak Russian, lol.

File: lang/de.json [hunk 4]
```diff
@@ -485,5 +485,9 @@
     "No, abort!": "Abbrechen",
     "Billing period": "Abrechnungsperiode",
     "Upgrading/Downgrading your server will reset your billing cycle to now. Your overpayed Credits will be refunded. The price for the new billing cycle will be withdrawed": "Das Upgrade/Downgrade Ihres Servers wird Ihre Abrechnungsperiode auf \"jetzt\" zurücksetzen. Ihre überzahlten Credits werden erstattet. Der Preis für die neue Abrechnungsperiode wird abgebucht.",
-    "Caution": "Achtung"
+    "Caution": "Achtung",
+    "You can not see your Referral Code": "Sie können Ihren Empfehlungscode nicht sehen",
+    "SERVER NAME": "SERVERNAMEN",
```
Comment (IceToast) [inline lang/de.json:490]: Why are those localizations full uppercase?

Comment (MrWeez) [reply lang/de.json:490]: > Why are those localizations full uppercase?

https://github.com/Ctrlpanel-gg/panel/blob/5827a143747f911b75ae8f35d1d50083667a1d93/themes/default/views/servers/settings.blade.php#L36

https://github.com/Ctrlpanel-gg/panel/blob/5827a143747f911b75ae8f35d1d50083667a1d93/themes/default/views/servers/settings.blade.php#L57

https://github.com/Ctrlpanel-gg/panel/blob/5827a143747f911b75ae8f35d1d50083667a1d93/themes/default/views/servers/settings.blade.php#L78

**Comment by IceToast** [comment]
Some conflicts need to be resolved, can be merged when they are

**Pull request closed without merging**